nginx/prod.nginx.conf

```diff
@@ -6,7 +6,7 @@ upstream api {
   server api:8081 weight=10 max_fails=3 fail_timeout=30s;
 }
 
-proxy_cache_path /nginx-cache levels=1:2 keys_zone=my_cache:30m inactive=60m use_temp_path=off max_size=4g;
+proxy_cache_path /nginx-cache levels=2:2 keys_zone=my_cache:300m inactive=1y use_temp_path=off max_size=4g;
```

Comment:
Why is this changed from `1:2` to `2:2`?

Reply:
For `16^5` it would need to be `1:2:2` or some permutation of that since only 1 or 2 are allowed. I'm not clear on the optimal number of files per folder so it's difficult for me to say what is enough. Anyways, if there are no performance downsides of having more folders than I don't see why we wouldn't go for the max of `2:2:2`.

Reply:
Ah right:
> levels of a cache: from 1 to 3, each level accepts values 1 or 2

There performance penalties for traversing more directories, but I'm hoping the machines are now using fast drives. Since `2:2` is not enough, `2:2:2` it is then.